src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -102,7 +102,7 @@ public RevisionsStorage(DocumentDatabase database, Transaction tx)
             {
                 Default = new RevisionsCollectionConfiguration
                 {
-                    MinimumRevisionAgeToKeep = TimeSpan.FromDays(45),
+                    MinimumRevisionsToKeep = 1024,
                     Disabled = false
```

Comment:
I think it would be advisable to set also MaximumRevisionsToDeleteUponDocumentUpdate to 10 * 1024, thoughts?

Reply:
Done